index.html

```diff
@@ -0,0 +1,246 @@
+				}
+			.cll {
+				width : 100% ;
+				height : 600px ;
+				}
+			.tb {
+				float : right ;
+				width : 40% ;
+				height : 100% ;
+				background-color : #4863A0 ;
+				}
+			.ar {
+				border-radius : 30px;
+				width : 100px ;
+				color : #4863A0 ;
+				padding : 5px ;
+				}
+			.ai {
+				padding : 40px ;
+				}
+			input {
+				padding : 5px ;
+				}
+			.ps {
+				width : 100% ;
+				height : 50px ;
+				}
+			.ff {
+				color : #4863A0 ;
+				}
+			.dg {
+				max-width : 100% ;
+				height : 170px ;
+				background-color : #2c3f33 ;
+				padding : 25px ;
+				}
+			.cole {
+				float : left ;
+				width : 25% ;
+				padding : 5px ;
+				}
+			.vls {
+				border-left : 1px solid grey ;
+				height : 160px ;
+				}
+			.wd {
+				float : left ;
+				width : 5% ;
+				height : 100% ;
+				padding : 5px ;
+				}
+			h2,p {
+				color : white ;
+				}
+			.vlc {
+				border-left : 1px solid grey ;
+				height : 105px ;
+				}
+			.dgs {
+				max-width : 100% ;
+				height : 80px ;
+				background-color : #2c3f33 ;
+				padding : 25px ;
+				}
+			.wit {
+				color : white ;
+				}
+			.colt {
+				float : left ;
+				width : 35% ;
+				padding : 5px ;
+				}
+			.dsi {
+				border : 2px solid #4863A0 ;
+				width : 130px ;
+				height : 30px ;
+				color : white ;
+				padding : 6px ;
+				border-radius : 30px ;
+				text-align : center ;
+				background-color : #4863A0 ;
+				}
+			.coly {
+				float : left ;
+				width : 5% ;
+				padding : 5px ;
+				}
+			.cold {
+				float : left ;
+				width : 55% ;
+				padding : 5px ;
+				}
+			.cntr {
+				position : absolute ;
+				top : 35% ;
+				left : 38% ;
+				transform : translate (-65%, -65%) ;
+				}
+			.blr {
+				filter : brightness(45%) ;
+				}
+		</style>
+	</head>
+	
+	<body>
+		<font face='Arial Unicode MS'>
+		<div class='bg'>
+			<div class='col'>
+				<img src='AVLancer-Header-Logo.png'>
+			</div>
+			<div class='cols'>
+				<div class='vl'></div>
+			</div>
+			<div class='cold'>
+				<div class='ds'>Are you a freelancer?</div>
+			</div>
+			<div class='coly'>
+				<br><font class='wit'>About</font>
+			</div>
+			<div class='coly'>
+				<br><font class='wit'>Contact</font>
+			</div>
+			<div class='cols'>
+				<div class='dsi'>Sign in</div>
+			</div>
+		</div>
+		<div class='res'>
+			<img src='home_banner.png' width='100%' height='100%' class='blr'>
+			<div class='cntr'><font size='6' class='wit'><center>WE'D LOVE TO<br><b>HEAR FROM YOU!</b></center></font></div>
+		</div>
+		<div class='ps'></div>
+		<div class='cll'>
+			<div class='tb'>
+				<form class='ai'>
+					<input type='text' value='Name*' size='28'><br><br>
+					<input type='email' value='Email*' size='28'><br><br>
+					<input type='text' value='Subject*' size='28'><br><br>
+					<textarea cols='31' rows='5'> Massage*</textarea><br><br>
+					<input type='submit' value='SUBMIT' class='ar'>
+				</form>
+			</div>
+			<center><font size='7'>Contact</font><br><br>
+			<table cellpadding='5'>
```

Comment:
Try using flexbox. Table is old school. https://www.w3schools.com/css/css3_flexbox.asp